.nvmrc

```diff
@@ -1 +1 @@
-18.0.0
\ No newline at end of file
+22.3.0
```

Comment:
Could we put a new line at the end of this file?

Reply:
Sure, what's the reason?

Reply:
According to the definition of a line in the POSIX standards https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206 it must end with a new line character.  That's why GitHub marks it with a red circle at the end of the file.  Technically this is a file with no lines in it.  I don't think it breaks NVM's reading of the file, but many tools don't handle this well.